requirements/requirements-python3.6.txt

```diff
@@ -123,21 +123,21 @@ future-fstrings==1.2.0
 future==0.18.2
 gcsfs==0.6.1
 google-ads==4.0.0
-google-api-core==1.16.0
+google-api-core==1.17.0
 google-api-python-client==1.8.0
 google-auth-httplib2==0.0.3
 google-auth-oauthlib==0.4.1
-google-auth==1.13.1
+google-auth==1.14.0
 google-cloud-automl==0.10.0
 google-cloud-bigquery-datatransfer==1.0.0
 google-cloud-bigquery==1.24.0
 google-cloud-bigtable==1.2.1
-google-cloud-container==0.4.0
+google-cloud-container==0.5.0
```

Comment:
I wouldn't have expected to see any changes here (and we shouldn't add these changes as part of this PR anyway). I wonder what's caused this though...

Reply:
When I ran the build the "generate requirements" step was failing. I'm honestly not sure why this was necessary either.